dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -172,6 +172,12 @@ func buildModules(srcPath, name, moduleFlag, version string, modules map[string]
 	}
 }
 
+func buildCSI(srcPath string) {
+	chdir(filepath.Join(srcPath, "integration/docker/csi"))
+	run(fmt.Sprintf("compiling csi"), "go", "build")
```

Comment:
I assume this would add the binary to `integration/docker/csi`, right? Then wherever we need this binary we need to point to this location, e.g. `integration/docker/entrypoint.sh`. Or we can also move this binary to where it's at now in Dockerfiles.